pyriemann/featureselection.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
```

Comment:
I would have put this function in pyriemann.classification

thoughts @qbarthelemy @sylvchev ?

Reply:
I'll put it in `preprocessing`,  as class distinctiveness measures the dispersion of input data and not the "discriminablity" of a classifier (it is classifier-independent). This is a nice supervised tool to evaluate the ability of the preprocessing part in a pipeline to correctly capture class-related information (e.g. impact of the choice of filtering, covariance estimator, etc). Thus preprocesing seems a good home for this code.

Reply:
@agramfort @qbarthelemy What do you think? Would it be okay to include this function in `preprocessing.py`?

Reply:
I have added the class_dis function at the end of the `preprocessing.py`.
I was not sure if I should define a new class for it. I would like to hear your opinion on that.

Reply:
Thank you all for the feedback! Okay, since @agramfort and @qbarthelemy two both suggested including this function in `classification.py`, let's put it that way! Thank you!